config/config.go

```diff
@@ -0,0 +1,95 @@
+package config
+
+import (
+	"fmt"
+	"reflect"
+	"strings"
+
+	"github.com/jeremywohl/flatten"
+	"github.com/mcuadros/go-defaults"
+	"github.com/mitchellh/mapstructure"
+	"github.com/spf13/viper"
+)
+
+// LoadConfig loads configuration into the given mapstructure (https://github.com/mitchellh/mapstructure)
+// from a config.yaml file and overrides with any values set in env variables
+func LoadConfig(config interface{}) {
```

Comment:
Let's call this just load. since config is covered in the package.

Reply:
yeah that makes sense.

Reply:
made the change